PeerioTechnologies/peerio-desktop
# fix: files search

#### Relevant info and issue/PR links 
files search fix
https://app.clubhouse.io/peerio/story/9941/desktop-searched-file-can-only-be-moved-to-root

minor related styling issues
https://app.clubhouse.io/peerio/story/9999/desktop-search-bar-clear-button-replace-and-alignment
https://app.clubhouse.io/peerio/story/2221/desktop-room-creation-form-text-fields-cut-off-the-first-letter

icebear PR
https://github.com/PeerioTechnologies/peerio-icebear/pull/256

#### Testing instructions  
* Files and MoveFileDialog search results are independent now, so one search bar should not affect the other. Moving files after a search should not be weird anymore.
* Inputs no longer use italics, so the text cutoff at the beginning is gone
* Search bar `X` icon has been swapped and the spacing adjusted

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
